libs/components/ag-grid/src/lib/styles/_base.scss

```diff
@@ -150,13 +151,19 @@ $modernDarkThemeCompact: map.merge($modernDarkThemeBase, $modernThemeCompact);
 .ag-theme-sky-data-entry-grid-modern-light-compact,
 .ag-theme-sky-data-grid-modern-dark-compact,
 .ag-theme-sky-data-entry-grid-modern-dark-compact {
+  @include ag-grid-extra.ag-grid-extra();
```

Comment:
I'm seeing a handful of style changes. Are the new visual tests screenshots desired? Looks like a new row is added underneath the grids...

Reply:
Added new visual tests.